script/bootstrap

```diff
@@ -4,4 +4,78 @@
 
 set -e
 
-cd "$(dirname "$0")/.."
\ No newline at end of file
+cd "$(dirname "$0")/.."
+
+export PYTHONUNBUFFERED=1
+export ANSIBLE_FORCE_COLOR=1
+
+script_self=$(dirname "$0")/bootstrap
+environment=$1
+
+if [ -z ${environment} ]
+then
+    echo "Error in ${script_self}:"
+    echo "  Parameter required: environment"
+    echo "  Example: ${script_self} vagrant"
+    exit 1
+fi
+
+playbook=${environment}.yml
+if [ ! -e ansible/${playbook} ]
+then
+    echo "Error in $(dirname "$0")/bootstrap:"
+    echo "  Playbook ansible/${playbook} does not exist, please check to make sure you're using the correct environment!"
+    exit 2
+fi
+
+inventory=inventory/${environment}.ini
+if [ ! -e ansible/${inventory} ]
+then
+    echo "Error in $(dirname "$0")/bootstrap:"
+    echo "  Inventory File ansible/${inventory} does not exist, please check to make sure you're using the correct environment!"
+    exit 3
+fi
+
+echo "Ansible Provisioning for environment:" ${environment}
+
+# This is hopefully a TEMPORARY section. I want to pre-bake a lot of this provisioning work into the actual box
+# we're using and not into the project itself.
+# <TemporaryProvisioning>
+echo "Provsioning the Server:"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "  1/5. Update apt"
+apt-get -y update
+
+if [ $(dpkg-query -W -f='${Status}' ansible 2>/dev/null | grep -c "ok installed") -eq 0 ];
+then
+    echo "  2/5. Add Ansible PPA"
+    apt-add-repository ppa:ansible/ansible
+
+    echo "  3/5. Install python2 and ansible"
+    apt-get -y install python python-dev python-pip ansible ntp
+else
+    echo "  2/5. Add Ansible PPA (Skipped, already installed)"
```

Comment:
It’s an if/then statement :)  Steps 2/3 repeat depending on which part of the if block it falls in. Unless I’m missing something?